mars/services/task/execution/ray/executor.py

```diff
@@ -224,6 +230,10 @@ async def execute_subtask_graph(
                 # TODO(fyrestone): Fetch(not get) meta object here.
                 output_meta_object_refs.append(meta_object_ref)
             context.update(zip(output_keys, output_object_refs))
+        self._stage_output_object_refs.append(stage_object_refs)
+        prev_progress = sum(self._stage_tile_progresses)
+        curr_tile_progress = self._tile_context.get_all_progress() - prev_progress
+        self._stage_tile_progresses.append(curr_tile_progress)
```

Comment:
Is the tile progress calculation should be at the beginning of the `execute_subtask_graph`?  The progress will be incorrect if `get_progress` while submitting the subtask graph in current implementation.

Reply:
Or can we calculate the tile progress when `get_progress` called?

Reply:
Yes, we should calculate tile progress before executing subtask graph and it's better than calculating every time in `get_progress`. Thanks for your reminder.